pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -118,21 +118,35 @@ func TestConfigPath(t *testing.T) {
 	const nondefaultPath = "/this/is/not/the/default/registries.conf"
 	const variableReference = "$HOME"
 	const rootPrefix = "/root/prefix"
+	oldHomeEnv, hasHomeEnv := os.LookupEnv("HOME")
+	err := os.Setenv("HOME", os.TempDir())
+	assert.Nil(t, err)
+	defer func() {
+		if hasHomeEnv {
+			os.Setenv("HOME", oldHomeEnv)
+		} else {
+			os.Unsetenv("HOME")
+		}
+	}()
+	var userRegistriesFile = filepath.FromSlash(".config/containers/registries.conf")
+	userRegistriesFilePath := filepath.Join(os.TempDir(), userRegistriesFile)
```

Comment:
Please use `ioutil.TempDir` to create a private directory for `$HOME` (and `defer os.RemoveAll()` to clean it up).

Using hard-coded paths directly in shared `TempDir` subjects users running the tests to attacks, e.g. by placing malicious symlinks.